### PR TITLE
Update developer.md

### PR DESCRIPTION
**Pull request by shansrin**
Updated the Developer Guide with the review comments.

### CODE REVIEW

File: docs/developer.md [hunk 1]
```diff
@@ -1,218 +1,110 @@
...
-
-## How to Build the Operator, Without Running Any Tests
-
-1. Clone and check out the `1.0` branch of the [GitHub repository](https://github.com/oracle/coherence-operator).
-
-   If you do not have a maven `settings.xml` file, create one.  If you
-   have one, make sure the following is included in your `default` profile.
-   All of the maven commands in this document are assumed to use this
-   `settings.xml` file.
+1. Clone and check out the operator version `1.0` from the [GitHub repository](https://github.com/oracle/coherence-operator).
```
Comment (tmiddlet2666) [inline docs/developer.md]: Should we specifically mention version 1.0 or the current version?

File: docs/developer.md [hunk 2]
```diff
@@ -1,218 +1,110 @@
...
 
-      4. Choose "Proceed to Checkout".
+      1. Clone the Oracle Docker Images git repository:
 
-      5. Create a Docker Id, or log in with it if you have one already.
+      `git clone git@github.com:oracle/docker-images.git`
 
-      6. Check the `I agree that my use of each program in this Content,
-         including any subsequent updates or upgrades...` box.
+      2. Change directory to `OracleCoherence/samples/122132-patch-for-k8s`.
```
Comment (tmiddlet2666) [inline docs/developer.md]: This should not be here. Please delete this line

File: docs/developer.md [hunk 3]
```diff
@@ -1,218 +1,110 @@
...
 
-      7. Check the `I acknowledge and allow Docker to share my personal
-         information linked to my Docker ID with this Publisher.` box.
+      3. Follow the steps [described here](https://github.com/oracle/docker-images/blob/master/OracleCoherence/samples/122132-patch-for-k8s/README.md) to create a Coherence 12.2.1.3.2 Docker image.
```
Comment (tmiddlet2666) [inline docs/developer.md]: delete this.

File: docs/developer.md [hunk 4]
```diff
@@ -1,218 +1,110 @@
...
 
-      8. Consider whether or not you want to check the `Please keep me
-         informed of products, services and solutions from this
-         Publisher` box.
+5. Tag the 12.2.1.3.2 image in the way it is required to build the operator.
```
Comment (tmiddlet2666) [inline docs/developer.md]: and this as we now have 12.2.1.3.2 image.

File: docs/developer.md (hunk 1 above)
Comment (shansrin) [reply docs/developer.md]: we can say current version for now. I can make that change.

**Pull request merged**